Makefile

```diff
@@ -106,6 +106,9 @@ test-ioloop: install
 test-servers: install
 	$(PYTHON) pyftpdlib/test/test_servers.py
 
+test-servers: install
+	$(PYTHON) pyftpdlib/test/test_demo_scripts.py
+
```

Comment:
There is already a make command with this name. You should rename this to `test-demos`